Add OPFS standalone article / Update API landing to "File System API" (#27440)
mdn/content: files/en-us/web/api/file_system_access_api/index.md

> **Note:** The different exceptions that can be thrown when using the features of this API are listed on relevant pages as defined in the spec. However, the situation is made more complex by the interaction of the API and the underlying operating system. A proposal has been made to [list the error mappings in the spec](https://github.com/whatwg/fs/issues/57), which includes useful related information.

> **Note:** Objects based on {{domxref("FileSystemHandle")}} can also be serialized into an {{domxref("IndexedDB API", "IndexedDB", "", "nocode")}} database instance, or transferred via {{domxref("window.postMessage", "postMessage()")}}.

### Origin private file system

The origin private file system (OPFS) is a storage endpoint provided as part of the File System Access API, which is private to the origin of the page and not visible to the user like the regular file system. It provides access to a special kind of file that is highly optimized for performance and offers in-place write access to its content.

> Read our [Origin private file system](/en-US/docs/Web/API/File_System_Access_API/Origin_private_file_system) for an understanding of how to use it.

Reviewer: ```suggestion
Read our [Origin private file system](/en-US/docs/Web/API/File_System_Access_API/Origin_private_file_system) for instructions on how to use it.
```
